.golangci.yml

```diff
@@ -1,3 +1,11 @@
+formatters:
+  enable:
+    - gofumpt       # Detects whether code was gofumpt-ed.
+    - goimports
+
+  exclusions:
+    generated: lax
```

Comment:
Also see my other comment; looks like it's happy with this removed as we don't (currently) have generated files, so we can remove these lines.